selfdrive/sensord/rawgps/rawgpsd.py

```diff
@@ -5,7 +5,7 @@
 import itertools
 import math
 import time
-import subprocess
+import subprocess as sp
```

Comment:
I haven't really seen this done in other projects, can you revert this to be consistent with the rest of openpilot? also found this cool site: https://import-as.github.io

Reply:
my bad, I am just used to use it this way, reverted